### PR TITLE
Pull Request for setup-mobile

### PR DESCRIPTION
**Pull request by anita00001**
This is a pull request for branch setup-mobile in the repository portfolio.
This branch contains a README.md file, an index.html file, a style.css file, an images folder to organize images, and linters setup files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  <link href='https://fonts.googleapis.com/css?family=Playfair Display' rel='stylesheet'>
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+  <header>
+    <nav class='toolbar'>
+      <a class="logo" href="#"></a>
+      <a class="menu"></a>
+    </nav>
+
```
Comment (tugsem) [inline index.html:18]: - [ ] Please close your header tag here at line 18 as typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+header {
+  display: flex;
+  flex-direction: column;
+  height: 440px;
+  width: 410px;
+}
+
+.toolbar {
+  width: 410px;
```
Comment (tugsem) [inline style.css:18]: - [ ] Please remove fixed width here as you are using flexbox to align items responsively according to the different screen sizes. See what I mean on screen size 375px(min):
- your toolbar does have a hamburger button but can't be seen: 
image

image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  color: white;
+  border: none;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  padding: 12px 42px;
+}
```
Comment (tugsem) [inline style.css:91]: - [ ] Kindly align this button using flexbox only. Make sure you have your button is centered horizontally. Also, add this background image after your button to make your site prettier and match the figma design:
image
add this background image: 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hi @anita00001 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️Github Flow
✔️Linters
✔️Descriptive pr
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sajid-munawar — APPROVED** [review]
### Hi @anita00001 , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.
 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @sajid-munawar in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**